docs/composedb/create-ceramic-app.mdx

```diff
@@ -0,0 +1,57 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Scaffold a new Ceramic app
+
+Get up and running quickly with a basic ComposeDB application with one command.
+
+You can easily create a simple ComposeDB starter project by using our CLI and running the following command:
+
+<Tabs
+  defaultValue="npm"
+  groupId="package-manager"
+  values={[
+    {label: 'npm', value: 'npm'},
+    {label: 'pnpm', value: 'pnpm'},
+    {label: 'yarn', value: 'yarn'},
+    {label: 'bun', value: 'bun'},
+  ]}>
+  <TabItem value="npm">
+
+```powershell
+npx create-ceramic-app
+```
+
+  </TabItem>
+  <TabItem value="pnpm">
+
+```powershell
+pnpx create-ceramic-app
+```
+
+  </TabItem>
+  <TabItem value="yarn">
+
+:::tip
+You need at least yarn 2.x to use the `yarn dlx` command. If you have an older version, upgrade it by running `yarn set version stable` and `yarn install`.
+
+Then you can run the following command to create a new Ceramic app using yarn 2.x
+:::
+
+```powershell
+yarn dlx create-ceramic-app
+```
+
+  </TabItem>
+  <TabItem value="bun">
+
+```powershell
+bunx create-ceramic-app
+```
+
+  </TabItem>
+</Tabs>
+
+This command will create a new directory with a basic ComposeDB application (a social app). It will clone the app from repository, install all dependencies, launch a local Ceramic node, a local GraphQL server and start the app.
```

Comment:
I'd suggest including the link to the repository if we mention that it will be cloned.